test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -55,21 +55,11 @@
     ('upsample_nearest3d_backward', datetime.date(9999, 1, 1)),
     ('_test_optional_float', datetime.date(9999, 1, 1)),
     ('aten::Int', datetime.date(2020, 1, 30)),
+    # We export some functions and classes for test_jit.py directly from libtorch.so,
+    # it's not important to have BC for them
+    ('_TorchScriptTesting.*', datetime.date(9999, 1, 1)),
```

Comment:
Nit: I guess `.*` is not needed.

Reply:
Yeah, it's search, but better be safe :)